### DISCUSSION TITLE
How can I combine Multiple CSS Files (for modules)

### DISCUSSION THREAD

**Discussion by musen34** [vercel/next.js#38905] [Help]
Hello,

I used sass on my project and when I get production build, it seems 8 css files from Chrome Network tab.

For example; I imported 6 modules to index.js (with dynamic method) and they have own scss files (sample> import styles from '../../styles/calculator.module.scss';). When I get production build, it seems 8 css files. I examined these files and I think 6 are for modules, one is globals.css (variables.css imported same file) and one is merged css (not all) file. 

How can I combine all in one css files?

Because this case affecting lighthouse performance test regarding to time to inteactive.. therefore I want to merge it.. 

Next 12.0.10
React 18

PS: I was not reproduce that  when I use the tailwind, it was one css file..

KR
Murat

**Comment by kimmy-wang** [comment]
same issue.

**Comment by taraskhvyl** [comment]
Still actual question
